Buttplug/Core/ButtplugJSONMessageParser.cs

```diff
@@ -89,10 +99,12 @@ public ButtplugMessage Deserialize(string aJsonMsg)
             return m;
         }
 
-        public static string Serialize(ButtplugMessage aMsg)
+        public string Serialize(ButtplugMessage aMsg)
         {
             var o = new JObject(new JProperty(aMsg.GetType().Name, JObject.FromObject(aMsg)));
-            return o.ToString(Formatting.None);
+            var a = new JArray(o);
+            _bpLogger.Trace($"Sent message: {a.ToString(Formatting.None)}");

```

Comment:
There's no sending happening here, we're just turning a message object into a json block, so the message is a little odd.